spring-cloud-marathon-connector/pom.xml

```diff
@@ -0,0 +1,51 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<artifactId>spring-cloud-marathon-connector</artifactId>
```

Comment:
Where do we specify the `marathon-connector` dependency for the `admin-mesos` project? I thought we need to explicitly specify the connector dependency into the `admin-mesos` pom.